src/input.rs

```diff
@@ -227,10 +229,10 @@ impl Action {
                 ::std::process::exit(0);
             },
             Action::IncreaseFontSize => {
-               ctx.change_font_size(1);
+               ctx.change_font_size(FONT_SIZE_STEP);
             },
             Action::DecreaseFontSize => {
-               ctx.change_font_size(-1);
+               ctx.change_font_size(FONT_SIZE_STEP);
```

Comment:
This needs to be negative, currently you're always increasing the font size.

Reply:
Nice catch, thanks!

Reply:
Done!